cmd/workload/historytest.go

```diff
@@ -42,6 +43,17 @@ type historyTestSuite struct {
 	tests historyTest
 }
 
+// errIsPrunedHistory returns true if an error is an rpc.Error with an error
+// code of 4444.
+func errIsPrunedHistory(err error) bool {
+	if err != nil {
+		if rpcErr := err.(rpc.Error); rpcErr != nil && rpcErr.ErrorCode() == 4444 {
```

Comment:
`rpcErr := err.(rpc.Error)` will crash if it doesn't satisfy the interface. You need to capture the second return value to check if it was OK.

Reply:
oof